environments/dev/dev-platform-hmcts-net.yml

```diff
@@ -29,7 +29,14 @@ A:
     record:
     - 10.145.15.250
     ttl: 300
+  - name: vh-user-api
+    ttl: 300
+    record:
+      - 10.145.15.250
 cname:
   - name: hmi-apim
     record: hmcts-dev.azurefd.net.
     ttl: 300
+  - name: vh-user-api
```

Comment:
why did you add the frontdoor entry? you just want the ingress controller

Reply:
I add frontdoor into public dns then ?

Reply:
Internal apis don’t go on front door